moped-editor/src/views/projects/projectView/ProjectFiles.js

```diff
@@ -264,7 +263,7 @@ const ProjectFiles = (props) => {
           id="file_description"
           name="file_description"
           value={props?.value ?? ""}
-          onChange={(e) => props.onChange(e.target.value.trim())}
```

Comment:
this was preventing users from being able to include spaces in their descriptions